lib/jekyll/drops/drop.rb

```diff
@@ -193,6 +193,32 @@ def merge!(other)
         end
       end
 
+      # Adds the resource behind this Drop as a dependency to whatever resource
+      # is rendering it. Paths are taken from the Liquid context available to
+      # the drop
+      #
+      # Returns nothing.
+      def link_dependency
+        site = @context.registers[:site]
+        requirer = @context.registers[:page]
```

Comment:
I don't see where `@context` is set. 😕

Reply:
IIRC `@context` is set by Liquid when it's rendering the Drop! https://github.com/Shopify/liquid/blob/v3.0.6/lib/liquid/template.rb#L176